src/pages/PassportRedemption/RedemptionSelectScreen.tsx

```diff
@@ -125,7 +126,14 @@ const PassportSelected = ({ setCurrentScreenView }: Props) => {
                 />
 
                 <SingleRewardInfo>
-                  <Text className="header">{sponsor.reward}</Text>
+                  <Text
+                    className="header"
+                    style={{
+                      fontSize: sponsor.reward.length > 20 ? '10px' : '12px',
```

Comment:
idk if character length is the right thing to be switching on here since each character has an inherently different width. Overflow should be the real determiner here. However, I recently made a backend change that helps solve this. Otherwise, I think it's better if each card has consistent style. cc @nanxiy @oliviasztanga 

Backend PR: https://github.com/sendchinatownlove/ruby/pull/223

Reply:
okies, let me just remove that file from my commits and we can figure that one out later tonight or smth